### DISCUSSION TITLE
List of reserved keywords

### DISCUSSION THREAD

**Discussion by adsharma** [modular/modular#3488] [General]
Is there a list of reserved keywords for mojo similar to this in python?

https://docs.python.org/3/library/keyword.html

**Comment by Moosems** [comment]
You mean like this?
https://docs.python.org/3/reference/lexical_analysis.html#keywords

No, not to my knowledge.

**Comment by adsharma** [comment]
Yes, that list is available as code from the link I shared:

https://github.com/python/cpython/blob/3.12/Lib/keyword.py#L18

Since mojo is a superset of python, I'll create a py2many PR with the list of keywords in the set `mojo` - `python3`.
Could use some review from this community.

**Reply by adsharma** [reply]
https://github.com/py2many/py2many/pull/651/files

Did I miss any documented keywords?
